### DISCUSSION TITLE
请问怎样可以在实体键盘手机上像GBoard那样完全隐藏软键盘

### DISCUSSION THREAD

**Discussion by top** [fcitx5-android/fcitx5-android#572] [General | 讨论]
请问是否有选项或者配置完全隐藏软键盘？
GBoard默认在有实体键盘的手机上不显示软键盘，这非常方便也简洁。
目前我在Fcitx5上试过最多只能将键盘高度降为10，不能再小了，也无法隐藏。
如果有任何建议非常欢迎，非常感谢。

**Comment by rocka** [comment]
- #106
- #571

别问了 在写了

**Comment by rocka** [comment]
基本功能都实现了，可以下载 CI Build 测试

https://jenkins.fcitx-im.org/job/android/job/fcitx5-android/
